docs/source/quickstart.mdx

```diff
@@ -0,0 +1,405 @@
+
+## 2. Create a gateway
+
+> You can also [clone our example gateway on GitHub](https://github.com/apollographql/supergraph-demo). The GitHub example is **not** identical to the gateway you create in this tutorial, but it demonstrates the same concepts.
+
+As mentioned in the [preface](#preface-federated-architecture), your federated supergraph is represented by a **gateway** that routes queries to various subgraphs. For this tutorial, we'll use some Apollo-hosted example services as our subgraphs, and we'll set up our own gateway in front of them.
+
+With the help of the `@apollo/gateway` library, [Apollo Server](https://www.apollographql.com/docs/apollo-server/) can act as our federated gateway.
+
+In a new Node.js project, first install the required libraries:
+
+```
+npm install apollo-server @apollo/gateway
+```
+
+Then paste the following into an `index.js` file as a minimal (not-yet-functional) gateway implementation:
+
+```js{4,7}:title=index.js
+const { ApolloServer } = require('apollo-server');
+const { ApolloGateway } = require('@apollo/gateway');
+
+const supergraphSchema = ''; // TODO!
+
+const gateway = new ApolloGateway({
+  supergraphSdl: supergraphSchema
+});
+
+const server = new ApolloServer({
+  gateway,
+  // Subscriptions are not currently supported in Apollo Federation
+  subscriptions: false
+});
+
+server.listen().then(({ url }) => {
+  console.log(`🚀 Gateway ready at ${url}`);
+}).catch(err => {console.error(err)});
+```
+
+This code demonstrates the basic flow for creating a gateway:
+
+1. We initialize an `ApolloGateway` instance and pass it the complete composed schema for our supergraph.
+    * _Note the `TODO`. We'll obtain the composed schema in the next section._
+
+2. We initialize an `ApolloServer` instance and pass it our gateway via the `gateway` option.
+    * _We also disable `subscriptions` because they aren't yet supported in Apollo Federation._
+
+3. We call `listen` on our server instance to begin listening for incoming requests.
+
+If we run this code as-is with `node index.js`, we get an error:
+
+```
+This data graph is missing a valid configuration. Syntax Error: Unexpected <EOF>.
+```
+
+That's because our `supergraphSchema` is currently empty! Next, we'll compose that schema.
+
+## 3. Compose the supergraph schema
+
+As a best practice, your gateway should _not_ be responsible for composing its own supergraph schema. Instead, a separate process should compose the schema and provide it to the gateway. This helps improve reliability and reduce downtime when you make changes to a subgraph.
+
+There are multiple ways to compose a supergraph schema from our subgraph schemas:
+
+* On our local machine using the Rover CLI (we'll start with this)
+* Via managed federation in Apollo Studio (we'll switch to this in a later section)
+
+Using managed federation is strongly recommended for production environments. We'll start with local composition just to get up and running.
```

Comment:
Maybe give people a link to skip forward to Part 2? IDK if part 2 can work as a standalone though.